### PR TITLE
User may connect their stripe account

### PR DESCRIPTION
**Pull request by zspencer**
Resolves https://github.com/wecohere/nourish.party/issues/79
Resolves https://github.com/wecohere/nourish.party/issues/10

### CODE REVIEW

**Comment by zspencer** [comment]
@bhaibel - This still need a refactor/alignment pass with the user interface flow you proposed, and probably some kind of feature tests of some kind (probably only a "given I've already connected a stripe account..." one, tbh)

File: app/assets/src/stylesheets/00-base/_variables.scss [hunk 1]
```diff
@@ -15,14 +15,16 @@ $base-z-index: 0;
...
+$sky-blue: #80d9ed;
 $purple: #57467B;
 $dark-gray: #333;
 $light-gray: #ddd;
 $white: #fff;
 
 // Font Colors
 $base-font-color: $dark-gray;
-$action-color: #80d9ed;
+$action-color: $sky-blue;
```
Comment (bhaibel) [inline app/assets/src/stylesheets/00-base/_variables.scss]: YAY VARIABLES

File: app/assets/src/stylesheets/01-atoms/notifications.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// Notification
+//
+// Some of text that says something the user should know about
+//
+// Markup:
+// <p class="notification --notice">This is what it is. You may or may not really care.</p>
```
Comment (bhaibel) [inline app/assets/src/stylesheets/01-atoms/notifications.scss]: YAY STYLEGUIDE BITS

File: app/models/registered_user.rb [hunk 3]
```diff
@@ -8,6 +8,7 @@ class RegisteredUser < ApplicationRecord
   has_many :project_memberships, foreign_key: :member_id
   has_many :projects, through: :project_memberships
   has_many :project_status_changes, foreign_key: :moderator_id
+  has_many :stripe_connections, foreign_key: :owner_id
```
Comment (bhaibel) [inline app/models/registered_user.rb]: Good call on making it a has_many.

File: app/views/stripe_connections/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<%= link_to(t('stripe_connections.create'), '/auth/stripe_connect', type: 'button') %>
```
Comment (bhaibel) [inline app/views/stripe_connections/index.html.erb]: I don't have a clear idea of where this view is used -- can you clarify or remove?

Blocking, but clarifying is enough to unblock.

Comment (zspencer) [reply app/views/stripe_connections/index.html.erb]: Leftover from when we had an index on StripeConnections. Deleting.

**Pull request merged**